resources/user-agents/browsers/firefox-focus/firefox-focus-android-1-0-on.json

```diff
@@ -37,6 +37,19 @@
             "Android_5_1", "Android_5_0",
             "Android"
           ]
+        },
+        {
+          "match": "Mozilla/5.0 (*Linux*Android?9) applewebkit* (*khtml*like*gecko*) Version/* Focus/#MAJORVER#.#MINORVER#* Chrome/* Mobile Safari/*",
```

Comment:
I added these as separate patterns because the `Android_9_0` platform pattern specifically requires a semicolon after the 9:  `*Linux*Android?9;*`, but these Firefox on Android 9 patterns don't use that.  I didn't want to change the platform pattern because then it opens the door for things like "Android 98" being caught as Android 9 UAs.